layers/+email/mu4e/packages.el

```diff
@@ -36,6 +38,9 @@
       (add-to-list 'mu4e-view-actions
                    '("View in browser" . mu4e-action-view-in-browser) t)
 
+      (when (configuration-layer/layer-usedp 'org)
+        (require 'org-mu4e))
```

Comment:
I think the `(with-eval-after-load 'org … ` cover all cases without the need of this part, no?